test/e2e/webhook-http-proxy-test

```diff
@@ -83,30 +83,18 @@ common_helm_args=()
 [[ "${TEST_WINDOWS-}" == "true" ]] && common_helm_args+=(--set targetNodeOs="windows")
 [[ -n "${NTH_WORKER_LABEL-}" ]] && common_helm_args+=(--set nodeSelector."$NTH_WORKER_LABEL")
 
-anth_helm_args=(
+aemm_helm_args=(
   upgrade
   --install
-  $CLUSTER_NAME-anth
-  $SCRIPTPATH/../../config/helm/aws-node-termination-handler/
-  --force
-  --namespace kube-system
+  $CLUSTER_NAME-aemm
+  $AEMM_DL_URL
   --wait
-  --set instanceMetadataURL="http://$AEMM_URL:$IMDS_PORT"
-  --set image.repository="$NODE_TERMINATION_HANDLER_DOCKER_REPO"
-  --set image.tag="$NODE_TERMINATION_HANDLER_DOCKER_TAG"
-  --set enableSpotInterruptionDraining="true"
-  --set enableScheduledEventDraining="true"
-  --set webhookURL="$WEBHOOK_URL"
-  --set webhookTemplate="\{\"Content\":\"[NTH][Instance Interruption] InstanceId: \{\{ \.InstanceID \}\} - InstanceType: \{\{ \.InstanceType \}\} - Kind: \{\{ \.Kind \}\} - Start Time: \{\{ \.StartTime \}\}\"\}"
-  --set webhookProxy="tcp://localhost:3128"
+  --namespace default
+  --set servicePort="$IMDS_PORT"
 )
-[[ -n "${NODE_TERMINATION_HANDLER_DOCKER_PULL_POLICY-}" ]] &&
-    anth_helm_args+=(--set image.pullPolicy="$NODE_TERMINATION_HANDLER_DOCKER_PULL_POLICY")
-[[ ${#common_helm_args[@]} -gt 0 ]] &&
-    anth_helm_args+=("${common_helm_args[@]}")
 
 set -x
-helm "${anth_helm_args[@]}"
+helm "${aemm_helm_args[@]}"
 set +x
 
```

Comment:
Need 

```
[[ ${#common_helm_args[@]} -gt 0 ]] &&
    aemm_helm_args+=("${common_helm_args[@]}")
```

to properly apply `targetNodeOs` and `nodeSelector`

Reply:
oops I thought I took that in the git conflict... I'll update

Reply:
fixed this and moved it to the webhook-secret test which looks like missed the train when these were added.